src/common/mfu_daos.c

```diff
@@ -2797,8 +2798,6 @@ static int serialize_recx_array(struct hdf5_args *hdf5,
             /* created attribute in HDF5 file with encoded
              * dataspace for this record extent */
             memset(attr_name, 0, ATTR_NAME_LEN);
-            memset(number_str, 0, ATTR_NAME_LEN);
-            memset(attr_num_str, 0, ATTR_NAME_LEN);
             path_len = snprintf(number_str, ATTR_NAME_LEN, "%d",
                                 (int)(*ak_index));
```

Comment:
Should this instead just use %lu and NOT cast to int, similar to the other instance you updated?